build-system/tasks/visual-diff/helpers.js

```diff
@@ -160,7 +160,7 @@ async function verifySelectorsVisible(page, testName, selectors) {
 async function waitForLoaderDots(page, testName) {
   const allLoaderDotsGone = await waitForElementVisibility(
```

Comment:
Optional: Perhaps rename this constant to `allLoadersGone`? And the outer function name to `waitForPageLoad`?

Reply:
Done